docs/milestone-1.md

```diff
@@ -0,0 +1,175 @@
+7. mev-boost must forward the [`SignedBlindedBeaconBlock`](#signedblindedbeaconblock) to all connected relays and attach the matching [`SignedMEVPayloadHeader`](#signedmevpayloadheader) using [`relay_proposeBlindedBlockV1`](#relay_proposeblindedblockv1) to inform the network of which relay created this payload.
+8. if an [`ExecutionPayloadV1`](https://github.com/ethereum/consensus-specs/blob/v1.1.6/specs/merge/beacon-chain.md#executionpayload) is returned, mev-boost must verify that the root of the transaction list matches the expected transaction root from the [`SignedBlindedBeaconBlock`](#signedblindedbeaconblock) before returning it to the BN.
+
+### required client modifications
+
+- consensus client must implement [blind transaction signing](https://hackmd.io/@paulhauner/H1XifIQ_t#Change-1-Blind-Transaction-Signing)
+
+## API Docs
+
+Methods are prefixed using the following convention:
+- `engine` prefix indicates calls made to the execution client. These methods are specified in the [execution engine APIs](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.5/src/engine/specification.md).
+- `builder` prefix indicates calls made to the mev-boost middleware.
+- `relay` prefix indicates calls made to a relay.
+
+### engine_forkchoiceUpdatedV1
+
+See [engine_forkchoiceUpdatedV1](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.5/src/engine/specification.md#engine_forkchoiceupdatedv1).
+
+### engine_getPayloadV1
+
+See [engine_getPayloadV1](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.5/src/engine/specification.md#engine_getpayloadv1).
+
+### builder_getPayloadHeaderV1
+
+#### Request
+
+- method: `builder_getPayloadHeaderV1`
+- params:
+  1. `payloadId`: `DATA`, 8 Bytes - Identifier of the payload build process
+
+#### Response
+
+- result: [`ExecutionPayloadHeaderV1`](https://github.com/ethereum/consensus-specs/blob/v1.1.6/specs/merge/beacon-chain.md#executionpayloadheader)
+- error: code and message set in case an exception happens while getting the payload.
+
+### builder_proposeBlindedBlockV1
+
+#### Request
+
+- method: `builder_proposeBlindedBlockV1`
+- params:
+  1. [`SignedBlindedBeaconBlock`](#signedblindedbeaconblock)
+
+#### Response
+
+- result: [`ExecutionPayloadV1`](https://github.com/ethereum/consensus-specs/blob/v1.1.6/specs/merge/beacon-chain.md#executionpayload)
+- error: code and message set in case an exception happens while proposing the payload.
+
+Technically, this call only needs to return the `transactions` field of [`ExecutionPayloadV1`](https://github.com/ethereum/consensus-specs/blob/v1.1.6/specs/merge/beacon-chain.md#executionpayload), but we return the full payload for simplicity.
+
+### relay_getPayloadHeaderV1
```

Comment:
Will MEV boost forward `relay_getPayloadHeaderV1`/`relay_proposeBlindedBlockV1` requests it receives to the relay?  The format of the spec here makes it look like these are available MEV boost endpoints, but I think they might be endpoints only the relay is required to serve.  

If these are only definitions for what the relay serves, I think it would make sense to have a clearer separation (in new section at the bottom here or maybe even splitting the relay API and mev-boost API into separate docs?)